debezium-schema-generator/pom.xml

```diff
@@ -38,7 +38,7 @@
     <dependency>
       <groupId>org.apache.maven</groupId>
       <artifactId>maven-plugin-api</artifactId>
-      <version>3.6.3</version>
+      <version>[3.8.4]</version>
```

Comment:
I don't think the brackets should be used here. Also as was mentioned in the Jira comment, could you please introduce a new property like `maven.version` that would be used by the enforcer plugin and scheme generator dependencies so it set in a single place?